deploy/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "flexy_cumulocity_connector_deploy",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "repository": "",
+  "author": "it-hms <iatr@hms.se>",
```

Comment:
Not a big concern, but shouldn't the author reflect the department?

```suggestion
  "author": "HMS Networks, MU Americas Solution Center",
```

Reply:
changed